tests/integration/test_subscribe.py

```diff
@@ -251,7 +251,65 @@ def matchesRecrawledDir(subdata):
                 if not sub['is_fresh_instance']:
                     continue
                 files = self.normWatchmanFileList(sub['files'])
-                if files == ab:
+                if self.fileListsEqual(files, ab):
+                    return True
+            return False
+
+        dat = self.waitForSub('myname', root=root,
+                              accept=matchesRecrawledDir)
+        self.assertNotEqual(None, dat)
+
+        # Ensure that we observed the recrawl warning
+        warn = None
+        for item in dat:
+            if 'warning' in item:
+                warn = self.decodeBSERUTF8(item['warning'])
+                break
+        self.assertRegexpMatches(warn, r'Recrawled this watch')
+
+    # TODO: Assimilate this test into test_subscribe when Watchman gets
+    # unicode support.
+    # TODO: Correctly test subscribe with unicode on Windows.
+    def test_subscribe_unicode(self):
+        if os.name == 'nt':
+            self.skipTest('This currently does not work on Windows.')
```

Comment:
No, this won't work either because the test skipping functionality just doesn't exist in Python 2.6's unittest. Just do what's done here: https://github.com/facebook/watchman/blob/master/tests/integration/WatchmanTestCase.py#L11